### PR TITLE
Allow core pages to serve without data

### PR DESCRIPTION
**Pull request by patcon**
Related to https://github.com/datamade/chi-councilmatic/pull/121

### CODE REVIEW

**Comment by patcon** [comment]
As far as I can tell, the whole chi site works without loading data with these changes made. Minor, but seemed a little easier on newcomers :)

File: councilmatic_core/views.py [hunk 1]
```diff
@@ -134,7 +135,10 @@ class CouncilMembersView(ListView):
     context_object_name = 'posts'
     
     def get_queryset(self):
-        return Organization.objects.get(ocd_id=settings.OCD_CITY_COUNCIL_ID).posts.all()
+        try:
+            return Organization.objects.get(ocd_id=settings.OCD_CITY_COUNCIL_ID).posts.all()
+        except ObjectDoesNotExist:
+            return dict()
         # return Post.objects.filter(organization__ocd_id=settings.OCD_CITY_COUNCIL_ID)
```
Comment (patcon) [inline councilmatic_core/views.py]: Wait. Can we just bring this back, but with an `OCD_DIVISION_ID`?

We don't currectly have that in `settings_jurisdiction.py`, but it's easily derivable from `OCD_JURISDICTION_ID`:

```py
OCD_JURISDICTION_ID = 'ocd-jurisdiction/country:us/state:il/place:chicago/government'
OCD_DIVISION_ID = '/'.join(OCD_JURISDICTION_ID.split('/')[:-1])
```

File: councilmatic_core/views.py [hunk 2]
```diff
@@ -300,8 +304,11 @@ def get_queryset(self):
     def get_context_data(self, **kwargs):
         context = super(EventsView, self).get_context_data(**kwargs)
         
-        aggregates = Event.objects.aggregate(Min('start_time'), Max('start_time'))
-        min_year, max_year = aggregates['start_time__min'].year, aggregates['start_time__max'].year
+        if Event.objects.exists():
```
Comment (patcon) [inline councilmatic_core/views.py]: I think this adds a query :/

**Comment by patcon** [comment]
ok, maybe hold off review. Wasn't being kind with queries before, and not sure how smart django is

**Comment by patcon** [comment]
OK, so this was a little messy before, and just quick, but I went back and revisited. I can now proudly put this out there for potential merge.

I've tested it to run with both a populated and empty database.

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,7 @@
 *.pyc
 __pycache__/*
 dist/*
+build/*
```
Comment (patcon) [inline .gitignore]: present when testing locally

File: councilmatic_core/views.py [hunk 4]
```diff
@@ -134,8 +134,7 @@ class CouncilMembersView(ListView):
     context_object_name = 'posts'
     
     def get_queryset(self):
-        return Organization.objects.get(ocd_id=settings.OCD_CITY_COUNCIL_ID).posts.all()
-        # return Post.objects.filter(organization__ocd_id=settings.OCD_CITY_COUNCIL_ID)
+        return Post.objects.all()
```
Comment (patcon) [inline councilmatic_core/views.py]: Is it proper to assume that only posts from the city will be in the database? If so, realized there is no need to filter, right?

**Comment by patcon** [comment]
cc: @cathydeng

**Comment by fgregg** [comment]
Okay, I think we are going to go with a different approach and have the template come with dummy data.

**Comment by patcon** [comment]
:+1:

**Pull request closed without merging**